ts/features/services/details/components/ServiceDetailsMetadata.tsx

```diff
@@ -26,6 +26,7 @@ type MetadataListItemAction = MetadataListItemBase & {
 
 type MetadataListItemInfo = MetadataListItemBase & {
   kind: "ListItemInfo";
+  label: string;
```

Comment:
Is this still needed? I don't see any usage in the code.

Reply:
Yes, its used by the `keyExtractor` ->  `keyExtractor={item => item.label}`